WORKSPACE

```diff
@@ -18,7 +18,7 @@ deps_gen()
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive", "http_file")
 
-NODE_VERSION = "22.11.0"
+NODE_VERSION = "22.14.0"
```

Comment:
This was needed to get `require()` working with `capnp-es` (which is ESM only) in our TS tests. I used `just update-deps` for Node.js, which I think is the correct approach (cc @anonrig ?)

Reply:
It's fine to ignore them for now. I'll take a look at once this lands. No need to block you.